cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -143,6 +149,8 @@ done
 
 echo "== default service account has token ${token_found} =="
 
+start_addon /etc/kubernetes/addons/namespace.yaml 100 10 default &
```

Comment:
Well, in this case, there shouldn't be --namespace on the command line. It's apparently ignored.